src/model/Organization.ts

```diff
@@ -132,4 +135,13 @@ export default class Organization extends Ressource {
   delete() {
     return super.delete(this.getLink("delete"));
   }
+
+  async addSubscription(payload:CreateSubscriptionPayloadType) {
+    const organizationId=this.id  
+    const organizationSubscription = new OrganizationSubscription({
+       subscription:{ organizationId, ...payload }
```

Comment:
Because we use only one time `organizationId`, don't need to create a variable, we can write directly
```
       subscription: { organizationId: this.id, ...payload }
  ```